src/sentry/utils/snuba.py

```diff
@@ -671,17 +671,15 @@ def get_related_project_ids(column, ids):
     return []
 
 
-def insert_raw(data):
-    data = json.dumps(data)
+def test_eventstream(data):
```

Comment:
Kind of seems to me like this is single purpose enough to just be contained within the `sentry.eventstream.snuba` package instead, I don't think anybody should be reaching into (re)use this?

Reply:
Yeah definitely.